openssl-sys/build.rs

```diff
@@ -7,10 +7,10 @@ use std::env;
 use std::default::Default;
 
 fn main() {
-    let target = env::var_string("TARGET").unwrap();
+    let target = env::var("TARGET").unwrap();
 
     if target.contains("android") {
-        let path = env::var_string("OPENSSL_PATH").ok()
+        let path = env::var("OPENSSL_PATH").ok()
```

Comment:
I think this needs to continue being var_string or it won't print properly below.